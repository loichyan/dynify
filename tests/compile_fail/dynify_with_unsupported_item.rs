#[dynify::dynify]
fn test() {}

#[dynify::dynify]
opaque_trait!();

fn main() {}
