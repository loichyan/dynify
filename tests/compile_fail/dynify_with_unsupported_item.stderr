error: non-trait item is not supported yet
 --> tests/compile_fail/dynify_with_unsupported_item.rs:2:1
  |
2 | fn test() {}
  | ^^^^^^^^^^^^

error: non-trait item is not supported yet
 --> tests/compile_fail/dynify_with_unsupported_item.rs:5:1
  |
5 | opaque_trait!();
  | ^^^^^^^^^^^^^^^^
